src/common/const.ts

```diff
@@ -68,6 +68,7 @@ export const FIXED_DEVICE_CLASS_ICONS = {
   power: "hass:flash",
   power_factor: "hass:angle-acute",
   signal_strength: "hass:wifi",
+  update: "mdi:update",
```

Comment:
I'm not too sure about this one, while the name matches, that icon does not make me think of updates, more of "jump forward" or something :man_shrugging: 

`mdi:arrow-up-bold-circle` is what we have used in the supervisor panel for this.

Reply:
personally, I think I prefer `mdi:package-up` over both those

Reply:
yeah for sure, since `mdi:arrow-up-bold-circle` is already used, I will change to it, already because of an unified look and feel 👍

Reply:
The supervisor can change as well :wink:

Reply:
OK, than ... what about `mdi:cellphone-system-update` ?
![image](https://user-images.githubusercontent.com/35783820/129491378-ba8855b0-7b5a-44c6-899e-b411af816192.png)

Reply:
That's looking more of an indication of downloading something

Reply:
I'm not really an UX expert, but for me all three of them would be ok in context of an "update available" sensor  🙈

Reply:
How about `mdi:checkbox-blank-circle` when no update, and `mdi:arrow-up-bold-circle` when there is one?

Reply:
I like `mdi:package-up` the most. It a good visualization and we may need to use `mdi:arrow-up-bold-circle` for something else in the future. Also credits for @ludeeus for suggesting `mdi:package` when there's no update.

Reply:
The switching icon logic is handled in `src/common/entity/binary_sensor_icon.ts`

Reply:
thx for the input, will have a look the upcoming days 👍

Reply:
![with_update](https://user-images.githubusercontent.com/15093472/129627764-3754af49-9c42-413b-b20c-3cefaa9d9324.png)
![without_update](https://user-images.githubusercontent.com/15093472/129627776-7b5cfc13-357f-4e9c-84d0-935d007478f6.png)